### PR TITLE
Update Commands.html

### PR DESCRIPTION
**Pull request by 0seastar0**

### CODE REVIEW

File: docs/Commands.html [hunk 1]
```diff
@@ -53,19 +53,19 @@ <h3 id=assemble>Command <code>assemble</code></h3>
...
 used to store data in the filesystem.
 
 <p>
-This also removes the <code>Data</code> directory.
+It also removes the <code>Data</code> directory.
 If a <code>DataOnDisk</code> directory exists,
-it creates a symbolic link to it, named <code>Data</code>.
-This makes it possible to use Shasta http server functionality
-and the Python API even after the binary data are 
+it creates a symbolic link named <code>Data</code>.
```
Comment (paoloczi) [inline docs/Commands.html:66]: This new wording does not say what the symbolic link points to.

File: docs/Commands.html [hunk 2]
```diff
@@ -89,25 +89,23 @@ <h3>Command <code>createBashCompletionScript</code></h3>
 
 <h3>Command <code>explore</code></h3>
 <p>
-This command starts Shasta in a mode in which it behaves as an
+This command starts Shasta in a mode that behaves as an
```
Comment (paoloczi) [inline docs/Commands.html:92]: Similar comment to a similar change yesterday. The new wording does not sound right to me.

File: docs/Commands.html [hunk 3]
```diff
@@ -158,8 +156,8 @@ <h3>Command <code>listConfigurations</code></h3>
 
 <h3 id=saveBinaryData>Command <code>saveBinaryData</code></h3>
 <p>
-This is used to save Shasta binary data.
-Binary data are stored in directory <code>Data</code> in the assembly directory.
+This command is used to save Shasta binary data.
+Shasta stores binary data in directory <code>Data</code> in the assembly directory.
 This command makes a copy on disk in <code>Data</code>.
```
Comment (paoloczi) [inline docs/Commands.html:161]: Actually the copy on disk is named `DataOnDisk`, not `Data`. This error was already there before these changes.

File: docs/Commands.html (hunk 2 above)
Comment (0seastar0) [reply docs/Commands.html:92]: it is correct :)

File: docs/Commands.html (hunk 3 above)
Comment (0seastar0) [reply docs/Commands.html:161]: Ok. I will update it with DataOnDIsk

File: docs/Commands.html (hunk 1 above)
Comment (0seastar0) [reply docs/Commands.html:66]: I will leave the original wording

**Pull request merged**